src/web/mjs/connectors/Manatoki.mjs

```diff
@@ -0,0 +1,59 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class Manatoki extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'manatoki82';
+        super.label = 'Manatoki';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://manatoki82.net';
+
+        this.path = '/comic/p%PAGE%';
+        this.pathMatch = /comic\/p(\d+)/;
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryChapter = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
+            resolve([...document.querySelectorAll('div.view-padding div > img')].map(img => img.src));
```

Comment:
try with this `[...document.querySelectorAll('div.view-padding p img')].map(img => img.src)`

Reply:
well actually they use lazyload on scroll

Reply:
the "p img" images are false images, not the ones for the chapter

Reply:
A bit squishy to changes, but this should do the trick for now
```javascript
scriptPages = `
    new Promise(resolve => {
        const images = [...document.querySelectorAll('div.view-padding p:not([class]) img')];
        resolve(images.map(image => Object.values(image.dataset).shift()));
    });
`;
```